content/en/contribute/_index.md

```diff
@@ -28,19 +46,18 @@ Go to the [Glossary GitHub repo](https://github.com/cncf/glossary/issues) issues
 
 ![issues](/images/how-to/howto-01.png)
 
-You'll see a variety of templates. To propose a new term in English, select "English Language Glossary Request."
+You'll see a variety of templates. To propose a new term in English, select "Request to add a new term (Default:English)."
 
-![templates](/images/how-to/howto-02.png)
+![templates](/images/how-to/english-issue-template-new.png)
 
-Add the word you're suggesting and hit "submit new issue." If you're just proposing a new term, you're done! To work on it, follow the next steps.
+Add the word you're suggesting, answer the two questions below, and hit "submit new issue." If you're just proposing a new term, you're done! To work on it, follow the next steps.
 
-![new issue](/images/how-to/howto-03.png)
+![new issue](/images/how-to/submit-issue.png)
```

Comment:
```suggestion
```
I think this screenshot can be removed :)

Reply:
nice catch :)